sdk/forms/index.html

```diff
@@ -1,70 +1,38 @@
+<!--
+
+# WARNING!
+
+The following HTML file is used for local development purposes only.
+It is not intended to be used in production or as a reference for a production implementation.
```

Comment:
Can't it still be a reference for production? obviously not localhost
```suggestion
It is not intended for production or as a reference for a production implementation.
```